### PR TITLE
feat: Initial frontend implementation 

### PR DESCRIPTION
**Pull request by Anlerkan**
### Description

- **Initial frontend implementation :tada:**

### CODE REVIEW

File: frontend/config/apiConfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "local": {
+      "BASE_URL": ""
+    },
+    "production": {
+      "BASE_URL": ""
+    }
+  }
+
```
Comment (Anlerkan) [inline frontend/config/apiConfig.json:9]: Unnecessary for now.

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Anlerkan) [inline frontend/public/index.html:27]: - Change title and description

File: frontend/src/home/page/form/PredictSentenceForm.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      <Button
+        isDisabled={query === ""}
+        shouldDisplaySpinner={isLoading}
+        type={"submit"}
+        customClassName={"blue predict-sentence-submit-button"}>
+        {"COMPUTE"}
+      </Button>
+      {error && <p className={"predict-sentece-form-error"}>{error}</p>}
+      {result.result === true && <Result result={result} />}
+      {result.result === false && <Result result={result} />}
```
Comment (Anlerkan) [inline frontend/src/home/page/form/PredictSentenceForm.tsx:50]: `{result.result ? <Result result={result}/> }`

File: frontend/src/core/root-app/RootApp.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import getPublicRouteComponents from "../route/publicRoutes";
+import HomePage from "../../home/page/HomePage";
+import getPublicFallbackRoutes from "../route/fallbackRoutes";
+
+
+function RootApp() {
+  return (
+    <BrowserRouter basename={"/"}>
+      <Route path={"/"} component={AppWithErrorBoundary} />
+    </BrowserRouter>
```
Comment (Anlerkan) [inline frontend/src/core/root-app/RootApp.tsx:14]: Use `<HashRouter>` instead of BrowserRouter

**Review by g3vxy — CHANGES_REQUESTED** [review]
frontend/.env shouldn't be in the repo @Anlerkan. Apart from that it seems good.

**Pull request merged**